docs/en/ingest-management/commands.asciidoc

```diff
@@ -31,31 +31,88 @@ NOTE: You might need to log in as a root user to run these commands.
 [[elastic-agent-enroll-command]]
 == elastic-agent enroll
 
-Enroll the {elastic-agent} in {fleet}.
+Enroll the {agent} in {fleet}.
+
+Use this command to enroll the {agent} in {fleet} without installing the
+agent as a service. You might want to do this if you installed
+the {agent} from a DEB or RPM package and plan to use systemd commands to
+start and manage the service. This command is also useful for testing
+{agent} prior to installing it.
 
-We recommend that you run this command as the root user because some
+TIP: To enroll an {agent} _and_ install it as a service, use the
+<<elastic-agent-install-command,`install` command>> instead.
+
+We recommend that you run the `enroll` command as the root user because some
 integrations require root privileges to collect sensitive data. This command
 overwrites the `elastic-agent.yml` file in the agent directory.
 
+This command includes optional flags to set up <<fleet-server,{fleet-server}>>.
+
+IMPORTANT: This command enrolls the {agent} in {fleet}; it does not start the
+agent. To start the agent, either <<start-elastic-agent,start the service>>, if
+one exists, or use the <<elastic-agent-run-command,`run` command>> to start the
+agent from a terminal.
+
+
 [discrete]
 === Synopsis
 
+//QUESTION: Which flags are relevant when running a Fleet
+//Server agent vs a regular agent? Which flags are required? (We should list
+//required flags first and they shouldn't be wrapped in square brackets). I'm
+//making some guesses here, but I'm not sure about the Ca and certificate
+//settings.
+
+//Also, should we include deprecated flags (like kibana-url) in the synopsis? I think
```

Comment:
I agree on not, here.  But maybe, we could mention 'kibana-url' in the trouble-shooting doc as a potential 'gotcha' that someone may be using an older, deprecated command.

Reply:
Also, we are putting the 'Fleet server' section first, wherever there is one... just because it is a firm requirement to have a FS before you can have 'other' subsequent Agents.  should we flip those 2 sections here?  Doing that in a separate follow on PR is fine, or skip it if you like it as is.

Reply:
`--kibana-url` was removed in 7.13, so its not included.

Reply:
> Also, we are putting the 'Fleet server' section first, wherever there is one... just because it is a firm requirement to have a FS before you can have 'other' subsequent Agents. should we flip those 2 sections here? Doing that in a separate follow on PR is fine, or skip it if you like it as is.

Jason pointed out during a different review that setting up Fleet Server will be a lot less common than installing/enrolling agents, so I'm going to leave this section as-is.